build-system/server/new-server/transforms/modules/test/esm-env-output.html

```diff
@@ -0,0 +1,3 @@
+<head>
+<script async="" src="https://cdn.ampproject.org/v0.js" nomodule=""></script>
+<script async="" src="http://localhost:8000/dist/v0.mjs" type="module"></script></head>
```

Comment:
Should there be a version of the transform (with a different test) that points to the CDN instead of `localhost` for `type="module"`?

Reply:
Yeah I would expect that the src would match for .js and .mjs

Reply:
oh right, missed that. should definitely be the cache domain

Reply:
Please take a look at the new PR changes.